### DISCUSSION TITLE
Course 1: Lesson 2: Concept 8: Q3 typo

### DISCUSSION THREAD

**Discussion by abulyomon** [OpenMined/courses#107] [Feedback] [answered]
"Correct - sometimes satisfying privacy is about forcing companies to **to** share or delete your data in a specific way or at a specific time."

**Comment by yemikifouly** [comment, marked as answer]
Thank you so much for bringing this to our attention Yaman! 🙌 I will pass it along so that the duplicate "to" can be removed. 🙂
